background-colors.suite/Scripts/randy2.script

```diff
@@ -0,0 +1,54 @@
+﻿//You know like an actual comment that tell you what the script is about
```

Comment:
@io-world Randy, just make sure you include the connection process within your script(s). These are meant to be self-contained solutions.